src/estimators/directive/index.ts

```diff
@@ -28,7 +28,8 @@ export default function (options?: {}): ComplexityEstimator {
   });
 
   return (args: ComplexityEstimatorArgs) => {
-    const values = getDirectiveValues(directive, args.field.astNode);
+    const astNode = (args.field && args.field.astNode) || {};
```

Comment:
If there is no astNode available on the field, we can return undefined right away and don't even need to get the directive values. That empty object is not a valid ast node object and only doesn't fail bcs. the type definition in graphql-js is incomplete. So this might break in a future graphql-js release.

Reply:
@ivome Thanks. I've added some test cases and also update my fix. Please check.